### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by lhy11009**
I fixed some typos.

### CODE REVIEW

File: include/aspect/mesh_refinement/isosurfaces.h [hunk 1]
```diff
@@ -113,14 +113,14 @@ namespace aspect
 
         /**
          * After cells have been marked for coarsening/refinement, apply
-         * additional criteria independent of the error estimate.
+         * additional criteria independent of the error estimation.
```
Comment (MFraters) [inline include/aspect/mesh_refinement/isosurfaces.h:116]: I think the original is more correct. Could you revert this change?

File: source/mesh_refinement/isosurfaces.cc [hunk 2]
```diff
@@ -372,22 +372,22 @@ namespace aspect
...
+                                              "checking whether the solutions of specific "
                                               "fields are within the ranges of values given. If these conditions "
                                               "hold, then an indicator is either put on or taken off "
                                               "in order to secure mesh refinement within the range of levels "
                                               "given. Usage of this plugin allows the user to put a conditional "
                                               "minimum and maximum refinement function onto fields that they "
                                               "are interested in."
                                               "\n\n"
-                                              "For now, only temperature and names of compositions are allowed as "
+                                              "For now, only temperature and compositional fields are allowed as "
```
Comment (MFraters) [inline source/mesh_refinement/isosurfaces.cc:383]: shouldn't we mention that the compositional field is entered by it's name (maybe provide an what the default names look like if no names are provided?)?

File: include/aspect/mesh_refinement/isosurfaces.h (hunk 1 above)
Comment (lhy11009) [reply include/aspect/mesh_refinement/isosurfaces.h:116]: done
